### DISCUSSION TITLE
Clicking a notification brings on a screen where the habit can't be checked

### DISCUSSION THREAD

**Discussion by ramnes** [iSoron/uhabits#1665] [Feature Requests]
Hey there, thanks for the wonderful app. :)

One thing that is driving me nuts is that every time I click a reminder notification rather than answering directly within the notification button, the habit stats page is opened. Since I can't answer for the habit on this stats page, I have to go back to the main screen, and now I can finally answer.

Am I missing something?

Edit: Probably related to #1121, although it's not the only solution I have in mind. One very easy fix would be that clicking a notification would bring you on the main screen rather than the details page.

**Comment by rkerssen** [comment]
Same here. It makes no sense, that reminders lead to the statistics page. They should lead to the main page, to the habit list, where one may check / answer the habit questions.

**Comment by chexmo** [comment]
+1 to this one
